src/pages/Search/SearchResult.tsx

```diff
@@ -9,21 +9,25 @@ import { mdLink, MimeTypes } from '../../utils';
 import { DriveIcon, ShortcutIcon, Table } from '../../components';
 import dayjs from 'dayjs';
 
-import './SearchResult.scss';
-
-interface IPropsSearchResult {
+const {fields} = config
 
+function escapeKeyword(keyword: string): string {
+  return keyword.
+  replaceAll('\\', `\\\\`).
```

Comment:
This might be too new, requires a browser updated in a month. Can you change it to `replace(/..../g, '...')`?